src/main/frontend/handler/block.cljs

```diff
@@ -286,7 +286,18 @@
                               result))))]
       (distinct (mapcat get-parents filtered-ref-blocks)))))
 
-(defn get-idx-of-order-list-block
+(defn convert-index
+  [idx delta]
+  (cond
+    (zero? delta) idx
+
+    (= delta 1)
+    (some-> (util/convert-to-letters idx) util/safe-lower-case)
+
+    :else
+    (util/convert-to-roman idx)))
+
+(defn get-raw-idx-and-delta-of-order-list-block
```

Comment:
Better use function [doc-string](https://clojuredocs.org/clojure.core/defn) here, instead of using a very long func name

Sample format:
https://github.com/logseq/logseq/blob/42ec1b0026632a694910f7814444ed3dbda1f7bc/src/main/frontend/handler/page.cljs#L125-L130